pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -206,6 +208,10 @@ func (a *ApplicationBackupController) createBackupLocationPath(backup *stork_api
 
 // handle updates for ApplicationBackup objects
 func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_api.ApplicationBackup) error {
+	if _, ok := backup.Annotations[storkVersion]; !ok {
```

Comment:
For backup and restore these should only be added for new or in progress ones

Reply:
Fixed. Added check to update the stork version, only for the backup/restore CR that are in initial, in-progress and pending state. Completed CR (success/failure), the stork version will not be updated.